### DISCUSSION TITLE
Question about doing a large translation update for fr_CA.

### DISCUSSION THREAD

**Discussion by daank-c** [dogecoin/dogecoin#3438] [Help needed]
Hey guys,

I'm currently working on updating the French Canadian file `bitcoin_fr_CA.ts` and I have a question before I go much further.

**How much is too much?** 

Where this is a new translation file, there is something like ~1,000 strings to be translated/brought up to date. This is going to be a lot to review. With a deadline coming up, should I maybe do this PR in pieces, or should I just hammer it all out? (I'm about halfway through, most of it is the same as `bitcoin_fr.ts`)

**Comment by patricklodder** [comment]
Even if it's not completely finished, it will be an improvement from what there is if you don't open a pull request. Reviewing 1000+ lines is a lot of work, and responding to reviews of 1000+ lines can take a lot of time too.

So maybe it's an idea to deliver a partial translation for this release, and finish it for the next (this can be done any time.)

**Reply by daank-c** [reply]
Okay, awesome. Sounds good.
